addons/uis/apps/ios.md

```diff
@@ -11,11 +11,48 @@ title: iOS openHAB App
 
 We provide a native iOS app for openHAB, compatible with iPhone and iPad.
 It uses the REST API of openHAB to render sitemaps of your openHAB installation.
-It also supports [myopenhab.org](http://www.myopenhab.org) including push notifications.
+
+**Features:**
+
+* View and control openHAB sitemaps, similar to the BasicUI
```

Comment:
I'd remove the second part. the iOS app is one of the equal options to view a sitemap.